src/SubjectPatternErrorParser.js

```diff
@@ -0,0 +1,2 @@
+module.exports = (subjectPatternError, subject, title) =>
+  subjectPatternError.replace('${subject}', subject).replace('${title}', title);
```

Comment:
What do you think about turning this into a generic module that can be used for all kinds of interpolations, possibly for other use cases as well?

```suggestion
module.exports = function formatMessage(message, values) {
  let formatted = message;
  Object.entries(values).forEach(([key, value]) => {
    formatted = formatted.replace(new RegExp(`{${key}}`, 'g'), value);
  });
  return formatted;
};
```

I added the `g` flag as well, if the variable is used multiple times.

Call site:

```js
formatMessage(subjectPatternError, {subject, title});
```

Reply:
sure man, I didn't do it generic as I didn't see how it could be reused in other cases. I'll update a pr in few minutes